erpnext/manufacturing/doctype/bom/bom.py

```diff
@@ -494,7 +494,7 @@ def get_exploded_items(self):
 					'image'			: d.image,
 					'stock_uom'		: d.stock_uom,
 					'stock_qty'		: flt(d.stock_qty),
-					'rate'			: d.base_rate,
+					'rate'			: flt(d.base_rate) / flt(d.conversion_factor),
```

Comment:
This will not work properly if the "Rate Of Materials Based On" is "Price List"

<img width="962" alt="Screenshot 2020-07-07 at 12 45 16 PM" src="https://user-images.githubusercontent.com/8780500/86734634-c2b7db80-c04f-11ea-8468-b8da9b02472d.png">